src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx

```diff
@@ -0,0 +1,131 @@
+
+## Prerequisites [#prereqs]
+
+This product has been tested and is compatible with VMware Tanzu OpsManager versions up to and including v3.0 and Tanzu Application Service 4.0.
+
+New Relic Dotnet Extension Buildpack for VMware Tanzu requires the following:
+
+* An active New Relic account with a license key. This is used to bind Dotnet apps to New Relic Dotnet agents.
+* To use multi-buildpacks in the app manifest, you need cf CLI v6.38 or later.
+For general information about adding buildpacks to manifests, see [Pushing an Application with Multiple Buildpacks](https://docs.pivotal.io/application-service/buildpacks/use-multiple-buildpacks.html) in the Cloud Foundry documentation.
+* To use the Dotnet HWC extension, you need HWC buildpack 3.0.3 or later.
+* To use the Dotnet Core extension, you need the dotnet core buildpack 2.1.5 or later.
+
+
+The following table provides version and version-support information about New Relic Dotnet Extension Buildpack for VMware Tanzu.
+
+<table>
+    <thead>
+    <tr>
+        <th>
+            Element
+        </th>
+
+        <th>
+            Details
+        </th>
+    </tr>
+    </thead>
+
+    <tbody>
+    <tr>
+        <td>
+            Tile version
+        </td>
+
+        <td>
+            1.1.12
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Release date
+        </td>
+        <td>
+            August 15, 2023
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Software component version
+        </td>
+        <td>
+            New Relic Dotnet Extension Buildpack 1.1.12
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Compatible Ops Manager version(s)
+        </td>
+        <td>
+            2.9.x, 2.10.x, and 3.0.x
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Compatible VMware Tanzu Application Service for VMs versions
+        </td>
+        <td>
+            2.10.x, 2.11.x, 2.12.x, 2.13.x, 3.0.x, and 4.0.x
+        </td>
+    </tr>
+    <tr>
+        <td>
+            BOSH stemcell version
+        </td>
+        <td>
+            Ubuntu Jammy
+        </td>
+    </tr>
+    <tr>
+        <td>
+            IaaS support
+        </td>
+
+        <td>
+            AWS, GCP, Azure, and vSphere
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+## Install the buildpack [#install]
+
+The New Relic Dotnet Extension Buildpack for VMware Tanzu can be installed via the tile in Ops Manager. Alternatively, you can extract the `.Pivotal` file and install individual extension buildpacks using the cf Command Line Interface (CLI) command `cf create-buildpack`.
+
+After you start monitoring your apps, you can set alerts based on any metrics that are collected by Dotnet agents using the New Relic alerting subsystem.
+
+The New Relic Dotnet Extension Buildpack for VMware Tanzu installs one or more of the following buildpacks depending on the tile configuration (total of 8 extension buildpacks):
```

Comment:
I'm not sure where this info belongs. It seems relelvant and the closest to an 'overview' type content. I can see it living near the section of content where we gesture to installing some buildpacks but not all, too